src/api/app/assets/javascripts/webui/datatables.js

```diff
@@ -13,7 +13,8 @@ var DEFAULT_DT_PARAMS = {
     info: "page _PAGE_ of _PAGES_ (_TOTAL_ records)",
     infoFiltered: ""
   },
-
+  responsive: true,
+  dom: 'ftpi',
```

Comment:
Why?

Reply:
hey @hennevogel 

if you look the documentation https://datatables.net/reference/option/dom, you will see that I'm changing the search elements. "ftpi" translates to:

f: filter
t: table
p: pagination
i: table summary

It controls which search elements are going to be displayed and in which order. So "ftpi" is different than "ftip" which would show the "table summary" before the "pagination". In the case I removed the "l" and reodered the "i" and "p" as we did in the Projects#index. I think the default is "fltip"

Reply:
I understand what you are doing. But why do you remove the length?

Reply:
> I understand what you are doing. But why do you remove the length?

why have it in first place?

Reply:
To have the option to have many things on one page. For overview I would guess.

Reply:
I don't understand, this has little to do with search.

Reply:
> To have the option to have many things on one page. For overview I would guess.

we had it because it was default in datatables. We didn't decide to have it.